keyboards/yellowbrickroad/info.json

```diff
@@ -0,0 +1,215 @@
+{
+    "keyboard_name": "yellowbrickroad",
+    "manufacturer": "3araht",
+    "url": "https://github.com/3araht/yellowbrickroad",
+    "maintainer": "3araht",
+    "bootloader": "rp2040",
+    "features": {
+        "midi": true,
+        "extrakey": true,
+        "encoder": true,
+        "bootmagic": false,
+        "console": false,
+        "mousekey": false,
+        "nkro": false,
+        "rgb_matrix": false
```

Comment:
off by default?

Reply:
I'll remove unused ones (console, mousekey, and nkro) and enable rgb_matrix here.